packages/terra-signature/src/Signature.scss

```diff
@@ -0,0 +1,8 @@
+.terra-Signature {
```

Comment:
with our use of css modules, I don't think we need to prefix the classname with terra nymore.